Source/Painting/SvgPaintServerFactory.cs

```diff
@@ -19,96 +17,42 @@ static SvgPaintServerFactory()
-                {
-                    if (value.StartsWith("url("))
-                    {
-                        var nextIndex = value.IndexOf(')', 4) + 1;
-                        var uri = new Uri(value.Substring(0, nextIndex), UriKind.RelativeOrAbsolute);
-                        value = value.Substring(nextIndex).Trim();
-                        servers.Add((SvgPaintServer)document.IdManager.GetElementById(uri));
-                    }
-                    // If referenced to to a different (linear or radial) gradient
-                    else if (document.IdManager.GetElementById(value) != null && document.IdManager.GetElementById(value).GetType().BaseType == typeof(SvgGradientServer))
-                    {
-                        return (SvgPaintServer)document.IdManager.GetElementById(value);
-                    }
-                    else if (value.StartsWith("#")) // Otherwise try and parse as colour
-                    {
-                        switch (CountHexDigits(value, 1))
-                        {
-#if !NETSTANDARD20
-                            case 3:
-                                servers.Add(new SvgColourServer((Color)_colourConverter.ConvertFrom(value.Substring(0, 4))));
-                                value = value.Substring(4).Trim();
-                                break;
-                            case 6:
-                                servers.Add(new SvgColourServer((Color)_colourConverter.ConvertFrom(value.Substring(0, 7))));
-                                value = value.Substring(7).Trim();
-                                break;
-#endif
-                            default:
-                                return new SvgDeferredPaintServer(value);
-                        }
-                    }
-#if !NETSTANDARD20
-                    else
-                    {
-                        return new SvgColourServer((Color)_colourConverter.ConvertFrom(value.Trim()));
-                    }
-#endif
-                }
+                colorValue = colorValue.Substring(nextIndex).Trim();
+                var fallbackServer = string.IsNullOrEmpty(colorValue) ? null : Create(colorValue, document);
 
-                if (servers.Count > 1)
-                {
-                    return new SvgFallbackPaintServer(servers[0], servers.Skip(1));
-                }
-                return servers[0];
+                return new SvgDeferredPaintServer(id, fallbackServer);
             }
-        }
 
-        private static int CountHexDigits(string value, int start)
-        {
-            int i = Math.Max(start, 0);
-            int count = 0;
-            while (i < value.Length &&
-                   ((value[i] >= '0' && value[i] <= '9') ||
-                    (value[i] >= 'a' && value[i] <= 'f') ||
-                    (value[i] >= 'A' && value[i] <= 'F')))
-            {
-                count++;
-                i++;
-            }
-            return count;
+#if NETSTANDARD20

```

Comment:
@mrbean-bremen
@wieslawsoltes

I am trying to rebase.
But I do not understand policy of netstandard2.0 support.(#509 )

If disable `SvgColourConverter`, `SvgPaintServerFactory` will hardly work.
So, is following correspondence right now ?

Reply:
Netstandard2 is not fully supported yet (especially `SvgColourConverter`), so this looks ok to me. 
@wieslawsoltes - can you also have a look?